Sources/Commands/SwiftTestTool.swift

```diff
@@ -356,6 +405,11 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
             option: parser.add(option: "--enable-code-coverage", kind: Bool.self,
                 usage: "Test with code coverage enabled"),
             to: { $0.shouldEnableCodeCoverage = $1 })
+
+        binder.bind(
```

Comment:
We can remove this as well since we're getting rid of showing code coverage for now.